tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,31 @@
+""" Handles logging functionality. """
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority,publisher_id,**log_payload):
```

Comment:
Done as per what was required, I wanted to know why handling caller = None was important in the log function itself?